### DISCUSSION TITLE
[Kavita] Add justify alignment and hyphenation in epub web reader

### DISCUSSION THREAD

**Discussion by orthodogz** [Kareadita/Kavita#3082] [Ideas]
### Idea Description

For now epubs exclusively  displayed with left alignment, which is atypical for most of epub readers, as they usually use justify alignment. I think that justification is more appropriate for books and much more pleasant to read. Hyphenation is used in most of epub readers too, and with justify alignment recreate real book experience fully, though it is harder to implement, for different languages especially, and I understand that this project is more about comics/manga then epub books. 

### Idea Category

Web UI

### Duration of Using Kavita

1 day

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
Can you give me an example of a reader that showcases the hyphenation effect? 

I didn't think justification made much an effect when most readers consume with the 1 column layout.

**Reply by orthodogz** [reply]
Kobo and apple books app heavily use it, though I dont know any open source reader that support hyphenation. 

Hyphenation helps to avoid lines where words are too widely spaced after justification space stretching.

**Reply by therobbiedavis** [reply]
This is an example of how to include hyphenation: https://css-tricks.com/almanac/properties/h/hyphenate/
